kernel/errno.c

```diff
@@ -44,10 +44,13 @@ static inline int *z_vrfy_z_errno(void)
 #include <syscalls/z_errno_mrsh.c>
 
 #else
+#ifndef CONFIG_PICOLIBC
```

Comment:
Probably better to use `#ifndef CONFIG_LIBC_ERRNO`.

Reply:
good catch. Will be fixed shortly.